lib/awesome_print/formatters/base_formatter.rb

```diff
@@ -123,18 +123,29 @@ def outdent
 
       def align(value, width)
         if options[:multiline]
-          if options[:indent] > 0
-            value.rjust(width)
-          elsif options[:indent] == 0
-            indent + value.ljust(width)
+          if options[:justify_hash]
+            align_justify(value, width)
           else
-            indent[0, indentation + options[:indent]] + value.ljust(width)
+            align_no_justify(value)
```

Comment:
something about this method name feels off, and it's doing more than just aligning / justifying, sort of-- indenting then justify. Is there a way to break it into those two steps?